rasa/shared/utils/cli.py

```diff
@@ -14,7 +26,10 @@ def print_color(*args: Any, color: Text) -> None:
         stream = AnsiToWin32(sys.stdout).stream
         print(output, file=stream)
     except ImportError:
-        print(output)
+        try:
+            print(output)
+        except BlockingIOError:
+            print_blocking(output)
 
 
 def print_success(*args: Any) -> None:
```

Comment:
"""Prints the given arguments in green color, indicating success.

    Args:
        *args: Any type of arguments to be printed.
    """
Add docstring

Reply:
This docstring is correct and valid but has nothing to do with the defect or the PR, so it doesn't belong here.